yt/funcs.py

```diff
@@ -1141,3 +1141,9 @@ def parse_h5_attr(f, attr):
 def issue_deprecation_warning(msg):
     from numpy import VisibleDeprecationWarning
     warnings.warn(msg, VisibleDeprecationWarning, stacklevel=3)
+
+def obj_length(v):
+    if iterable(v):
+        return len(v)
+    else:
+        return 0
```

Comment:
could you put a comment here noting that instead of this returning 1 if it's not iterable, it returns 0?  and maybe it should be `False` instead?